core/src/test/resources/QualificationExpectations/nested_type_expectation.csv

```diff
@@ -1,2 +1,2 @@
-App Name,App ID,Recommendation,Estimated GPU Speedup,Estimated GPU Duration,Estimated GPU Time Saved,SQL DF Duration,SQL Dataframe Task Duration,App Duration,GPU Opportunity,Executor CPU Time Percent,SQL Ids with Failures,Unsupported Read File Formats and Types,Unsupported Write Data Format,Complex Types,Nested Complex Types,Potential Problems,Longest SQL Duration,NONSQL Task Duration Plus Overhead,Unsupported Task Duration,Supported SQL DF Task Duration,Task Speedup Factor,App Duration Estimated
-Spark shell,local-1629446106683,Not Recommended,1.05,16729.33,968.66,1453,6475,17698,1453,27.76,"","","",array<struct<city:string;state:string>>;map<string;map<string;string>>,array<struct<city:string;state:string>>;map<string;map<string;string>>,NESTED COMPLEX TYPE,1453,11904,0,6475,3.0,false
+App Name,App ID,Recommendation,Estimated GPU Speedup,Estimated GPU Duration,Estimated GPU Time Saved,SQL DF Duration,SQL Dataframe Task Duration,App Duration,GPU Opportunity,Executor CPU Time Percent,SQL Ids with Failures,Unsupported Read File Formats and Types,Unsupported Write Data Format,Complex Types,Nested Complex Types,Potential Problems,Longest SQL Duration,NONSQL Task Duration Plus Overhead,Unsupported Task Duration,Supported SQL DF Task Duration,Task Speedup Factor,App Duration Estimated,Unsupported Execs,Unsupported Expressions
+Spark shell,local-1629446106683,Not Recommended,1.05,16729.33,968.66,1453,6475,17698,1453,27.76,"","","",array<struct<city:string;state:string>>;map<string;map<string;string>>,array<struct<city:string;state:string>>;map<string;map<string;string>>,NESTED COMPLEX TYPE,1453,11904,0,6475,3.0,false,,
```

Comment:
hmm normally I would have expected empty strings to be here... Maybe its ok but I think we did that so csv was parsed properly... we should file a follow on issue

Reply:
issue filed: https://github.com/NVIDIA/spark-rapids-tools/issues/96